src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -248,10 +248,16 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
 
   const { ownedVenues } = useOwnedVenues({});
 
-  const venueSpaces = ownedVenues.map(({ name, template }) => ({
-    name,
-    template,
-  }));
+  const backButtonOptionList = ownedVenues.filter(({ id, name, template }) => {
```

Comment:
LOL, can't say I read `backButtonOptionList` better than `venueSpaces`, but at least it's less abstract :D